File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/MailBlockRspamd.scala

package org.openapitools.client.model


case class MailBlockRspamd (
    /* The sender email address.  Pass this value as `email` to `POST /mail/blocks/delete` to delist the sender. */
    _from: String,
    /* The suspicious subject pattern that triggered the block. */
    _subject: String
)
object MailBlockRspamd {
    def toStringBody(var_from: Object, var_subject: Object) =
        s"""
        | {
        | "from":$var_from,"subject":$var_subject
        | }
        """.stripMargin
}

File: openapi-client/javascript-closure-angular/API/Client/MailBlockRspamd.js
goog.provide('API.Client.MailBlockRspamd');

/**
 * A block pattern sourced from the rspamd database.  Represents a sender whose recent messages contained suspicious subject lines (e.g. containing relay/proxy strings) repeated more than 4 times in the last 3 days. The `from` address can be passed to `POST /mail/blocks/delete` to delist it.
 * @record
 */
API.Client.MailBlockRspamd = function() {}

/**
 * The sender email address.  Pass this value as `email` to `POST /mail/blocks/delete` to delist the sender.
 * @type {!string}
 * @export
 */
API.Client.MailBlockRspamd.prototype.from;

/**
 * The suspicious subject pattern that triggered the block.
 * @type {!string}
 * @export
 */
API.Client.MailBlockRspamd.prototype.subject;

